features/FEATURE_BLE/targets/TARGET_Cypress/TARGET_CYW43XXX/TARGET_PSOC6/cy_bt_cordio_cfg.cpp

```diff
@@ -0,0 +1,115 @@
+/*******************************************************************************
+* \file cy_bt_cordio_cfg.cpp
+* \version 1.0
+*
+*
+* Low Power Assist BT Pin configuration implementation.
+*
+********************************************************************************
+* \copyright
+* Copyright 2019-2020, Cypress Semiconductor Corporation.  All rights reserved.
```

Comment:
Can you align the license as other files from PSOC6 ? This is not a license we can accept.